models/Activity.ts

```diff
@@ -173,16 +171,26 @@ export class ActivityModel extends Stream<Activity, ActivityFilter>(ListModel) {
   }
 
   @toggle('downloading')
-  getQuestionnaire(name: string) {
-    const res = this.client.get<Questionnaire>(
-      `${this.baseURI}/${name}/questionnaire`,
-    );
-    return res;
+  async getQuestionnaire(activity = this.currentOne.name) {
+    try {
+      const { body } = await this.client.get<Questionnaire>(
+        `${this.baseURI}/${activity}/questionnaire`,
+      );
+      const questionnaire = body!.extensions.map(
+        v => JSON.parse(v.value) as Question,
+      );
+      return questionnaire;
+    } catch (error) {
+      return [];
+    }
```

Comment:
```suggestion
      const { body } = await this.client.get<Questionnaire>(
        `${this.baseURI}/${activity}/questionnaire`,
      );
      const questionnaire = body!.extensions.map(
        v => JSON.parse(v.value) as Question,
      );
      return questionnaire;
```
尽量不要吞异常，让上层去 `catch`。